test_scripts/Resumption/InteriorVehicleData/005_InteriorVD_resumption_all_modules_with_moduleId_unexpected_disconnect.lua

```diff
@@ -0,0 +1,85 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: Successful resuming of interior vehicle data after transport disconnect
+--  in case GetInteriorVehicleData was requested for all modules and with moduleId
```

Comment:
@GetmanetsIrina I would propose next rephrasing of the test script to avoid confuse
```suggestion
--  in case app was subscribed to some of modules (one of each module type)
```